web/app/themes/clarity/src/components/c-input-container/on-change.js

```diff
@@ -0,0 +1,7 @@
+export default (function ($) {
```

Comment:
Just checking if this is part of the search enhancement and if it should go in with this.

Reply:
Good question. 
This is to do with supporting radio elements on the search form. I've added it to compliment the form builder api which we will be using in the next couple of weeks so left it in for that. Happy to walk through what I did tomorrow, if you're free buddy.